src/main/java/io/github/fusionflux/portalcubed/mixin/DoorBlockMixin.java

```diff
@@ -0,0 +1,71 @@
+package io.github.fusionflux.portalcubed.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyArg;
+
+import com.llamalad7.mixinextras.injector.wrapoperation.Operation;
+import com.llamalad7.mixinextras.injector.wrapoperation.WrapOperation;
+
+import io.github.fusionflux.portalcubed.content.door.ChamberDoorBlock;
+import io.github.fusionflux.portalcubed.content.door.LockingChamberDoorBlock;
+import net.minecraft.world.level.block.DoorBlock;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.properties.Property;
+
+@Mixin(DoorBlock.class)
+public class DoorBlockMixin {
+	@SuppressWarnings("rawtypes")
+	@WrapOperation(
+			method = "<init>",
+			at = @At(
+					value = "INVOKE",
+					target = "Lnet/minecraft/world/level/block/state/BlockState;setValue(Lnet/minecraft/world/level/block/state/properties/Property;Ljava/lang/Comparable;)Ljava/lang/Object;",
+					ordinal = 1
+			)
+	)
+	private Object registerStatePropertyDefault(BlockState instance, Property property, Comparable comparable, Operation<Object> original) {
+		if ((Object) this instanceof LockingChamberDoorBlock)
+			return original.call(instance, LockingChamberDoorBlock.STATE, LockingChamberDoorBlock.State.CLOSED);
+		return original.call(instance, property, comparable);
+	}
+
+	@SuppressWarnings("rawtypes")
```

Comment:
don't suppress rawtypes, fix the types

Reply:
It won't fit the method method signature then

Reply:
put it on multiple lines